packages/commons-server/src/libs/server/admin-endpoint.ts

```diff
@@ -1,43 +1,88 @@
 import { Express } from 'express';
-
 const adminApiPrefix = '/mockoon-admin';
 
 /**
  * Expose two endpoints to purge the state
  * PURGE /mockoon-admin/state
  * POST /mockoon-admin/state/purge
+ * POST /mockoon-admin/state/setGlobalVars
+ * POST /mockoon-admin/state/purgeGlobalVars
  *
  * @param app
  * @param statePurgeCallback
+ * @param setGlibalVariables
+ * @param purgeGlobalVariables
  */
-const createStateEndpoints = (app: Express, statePurgeCallback: () => void) => {
+const createStateEndpoints = (
+  app: Express,
+  statePurgeCallback: () => void,
+  setGlobalVariables: (key: string, value: any) => void,
+  purgeGlobalVariables: () => void
+) => {
   const stateEndpoint = `${adminApiPrefix}/state`;
   const purgeHandler = (req, res) => {
     statePurgeCallback();
     res.send({
       response: 'State purged successfully.'
     });
   };
-
+  const setGlobalVar = (req, res) => {
+    try {
+      const { key, value } = req.body; // Destructure key and value from req.body
+      if (key && value) {
+        setGlobalVariables(key, value); // Set global variables
+        res.send({
+          message: `Global variable ${key} has been set to ${value}.`
+        });
+      } else {
+        throw new Error('Key or value missing from request'); // Throw error if key or value are missing
+      }
+    } catch (err) {
+      console.error('Error handling request:', err);
+      res.status(400).send({ message: 'Invalid JSON or missing key/value' });
+    }
+  };
+  const purgeGlobalVars = (req, res) => {
+    try {
+      purgeGlobalVariables();
+      res.send({
+        message: 'Global variables have been purged.'
+      });
+    } catch (err) {
+      console.log(err);
```

Comment:
the console.error will be lost, you should use the logger instead. But I think the call response is enough and we could remove this.